cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -277,6 +279,14 @@ public void testDeleteAll() throws Exception {
     Assert.assertTrue(streamAdmin.exists(myStream));
     assertResponseCode(200, deleteNamespace(OTHER_NAME));
     Assert.assertFalse(streamAdmin.exists(myStream));
+
+    // Create the namespace again and deploy the application containing schedules.
+    // Application deployment should succeed.
+    assertResponseCode(200, createNamespace(OTHER_NAME));
+    HttpResponse response = deploy(AppForUnrecoverableResetTest.class, Constants.Gateway.API_VERSION_3_TOKEN,
+                                   OTHER_NAME);
+    Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+    assertResponseCode(200, deleteNamespace(OTHER_NAME));
```

Comment:
do we delete the schedules when we delete an app? or only during unrecoverable reset ? it may be better to delete the app with schedules and test re-deployment of that app, rather than adding the test here in deleteAll, right ?